src/main/java/core/basesyntax/Reader.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class Reader {
+    public List<String> readCsvFileWithData(Path pathFile) {
+        List<String> result;
+        try {
+            result = Files.readAllLines(pathFile);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file " + pathFile, e);
+        }
+        return result;
```

Comment:
The method `readCsvFileWithData` is well-named following the convention and is concise. However, consider if there's a need to process the CSV data within this method or if it should only be responsible for reading the file. If CSV processing is required, the method should be divided accordingly.